### ISSUE TITLE
Invalid save path

### ISSUE THREAD

**Issue by wogam** [hayden-fr/ComfyUI-Model-Manager#17]
Whenever downloading any model into /loras/ I get the error. Folder exists.

**Comment by cdb-boop** [comment]
Yes, `/loras/` is just the type of model, not an actual folder. The `/0/` after is the index in the list of directories ComfyUI is using for each model type. You can specify more directories in `extra_model_paths.yaml`. Currently the directory must pre-exist.

Are you using `/loras/0`? Can you not do `/loras/0/my/sub/directory` to download or move the model? Can you share the command line error?
